v2/ticker_service.go

```diff
@@ -118,9 +125,14 @@ func (s *ListPriceChangeStatsService) Do(ctx context.Context, opts ...RequestOpt
 		method:   http.MethodGet,
 		endpoint: "/api/v3/ticker/24hr",
 	}
+
 	if s.symbol != nil {
 		r.setParam("symbol", *s.symbol)
+	} else if s.symbols != nil {
+		s, _ := json.Marshal(s.symbols)
+		r.setParam("symbols", s)
```

Comment:
Sorry, it should be:
```suggestion
		r.setParam("symbols", s.symbols)
```

Reply:
yeah, updated